src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -50,7 +60,29 @@ export class RelayFeeCalculator {
       this.capitalCostsPercent >= 0 && this.capitalCostsPercent <= 100,
       "capitalCostsPercent must be between 0 and 100 percent"
     );
+    this.capitalCostsConfig = config.capitalCostsConfig ? JSON.parse(config.capitalCostsConfig) : {};
+    for (const token of Object.keys(this.capitalCostsConfig)) {
+      RelayFeeCalculator.validateCapitalCostsConfig(this.capitalCostsConfig[token]);
+    }
+  }
+
+  static validateCapitalCostsConfig(_capitalCosts: unknown) {
```

Comment:
when you do something like this, you can `static validateCapitalCostsConfig(_capitalCosts: unknown) as { [token: string]: CapitalCostConfig }` and it will "prove" to typescript that if this returns the type is correct ( i think) @mrice32

Reply:
I ended up removing the `unknown` type as its not needed anymore